source/core/parameters_lagrangian.cc

```diff
@@ -110,10 +357,6 @@ namespace Parameters
                           Patterns::Selection("uniform|non_uniform"),
                           "Choosing insertion method. "
                           "Choices are <uniform|non_uniform>.");
-        prm.declare_entry("n total",
-                          "1",
-                          Patterns::Integer(),
-                          "Total number of particles");
```

Comment:
Yes, for insertion of polydispersed systems, I decided to define the number of particles of each type right after defining the particle type information. I thought it would be easier for users.